src/core_plugins/kibana/public/home/components/_recently_accessed.scss

```diff
@@ -0,0 +1,32 @@
+@include euiBreakpoint('xs', 's') {
+  .homRecentlyAccessed__separator {
+    display: none;
+  }
+}
+
+.homRecentlyAccessed__item {
+  overflow: hidden;
+  max-width: 300px;
+
+  // SASSTODO: Figure out how to not use eui class here
+  // ... tooltips don't pass classnames to the anchor and
+  // ... this is the only way to get truncation to work
```

Comment:
I believe we allow this. `anchorClassName`. It's just not documented in the auto-docs.